src/utils/devtools.ts

```diff
@@ -15,10 +21,10 @@ const DEVTOOLS = Symbol()
  * const state = proxy({ count: 0, text: 'hello' })
  * const unsub = devtools(state, 'state name')
  */
-export function devtools<T extends object>(proxyObject: T, name?: string) {
-  let extension: any
+export function devtools<T extends object>(proxyObject: T, name = '') {
```

Comment:
Does the original one work? No?
```suggestion
export function devtools<T extends object>(proxyObject: T, name?: string) {
```

Reply:
No it does not! It's not aligned with extension.connect!

Reply:
Alright!